larq_zoo/training/knowledge_distillation/multi_stage_training.py

```diff
@@ -106,7 +106,11 @@ class LarqZooModelTrainingPhase(TrainingPhase):
     output_matching_train_teacher: bool = Field(False)
 
     @Field
-    def loss(self) -> Optional[Union[tf.keras.losses.Loss, str]]:
+    def loss(
+        self,
+    ) -> Optional[
+        Union[Sequence[Union[tf.keras.losses.Loss, str]], tf.keras.losses.Loss, str]
+    ]:
```

Comment:
Since the annotation is defined on the superclass, how do you feel about completely removing it here?
```suggestion
    def loss(self):
```